frontend/src/components/Header/Header.js

```diff
@@ -17,6 +21,13 @@ const Header = ({
     }
   };
   const loginOrCreateButton = () => {
+    if (user && match.params.id) {
```

Comment:
This feels a little hacky, as there might be other pages that use ids in the URL. I'm also curious if there's a more elegant way to handle this problem than forcing a reload, but not sure off the top of my head what might be quick to implement. But I don't necessarily want to block stuff getting merged at this point.

Reply:
From the user's perspective, I don't know that it would feel unnatural or unexpected for the page to reload if they're going from editing a recipe to creating an entirely new one. I feel like almost any other solution would require many more lines of code so idk....

Also I think the only other pages that have ids in the url are the individual recipe pages, and the CreateRecipeForm component would have to load up anyway when navigating there anyway, right? Will have to look into this more tomorrow.